source/plugin/project/version-numbers.rst

```diff
@@ -58,9 +57,9 @@ Timestamp
 
 This is a less commonly used versioning scheme, but it is also a very simple one, as it does not contain any additional
 information in the version numbers. It does not matter which separators you use (if any) or which date and time parts
-you include. However we recommend you to order date and time parts in the descending order of duration like
-``YYYY-MM-DD``. If you sort your files alphabetically, please make sure that the order would be the same as the version
-order. It is also possible to add a build qualifier or a build number.
+you include. However, we recommend ordering date and time parts in descending order of duration like ``YYYY-MM-DD``. 
+If you sort your files alphabetically, please make sure that the order would be the same as the versionorder. It is 
```

Comment:
Why is "versionorder" one word now?

Reply:
BEcause fuck VIM :stuck_out_tongue: Lemme fix that real quick...